<ui version="4.0" >
 <author>&lt;aumuell@reserv.at></author>
 <class>OrganizeCollectionDialogBase</class>
 <widget class="QWidget" name="OrganizeCollectionDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Organize Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" >
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="folderLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Base directory under which to put files</string>
       </property>
       <property name="text" >
        <string>C&amp;ollection Folder:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>folderCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="folderCombo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabPosition" >
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>File Naming Scheme</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QCheckBox" name="filetypeCheck" >
         <property name="toolTip" >
          <string>If checked, create a directory hierarchy using the filename extension.</string>
         </property>
         <property name="text" >
          <string>Group b&amp;y File Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="initialCheck" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>If checked, introduce another directory hierarchy for the artists' initials.</string>
         </property>
         <property name="text" >
          <string>Group &amp;by Artist's Initial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="customschemeCheck" >
         <property name="toolTip" >
          <string>If checked, use a custom format string for naming the files in the collection</string>
         </property>
         <property name="text" >
          <string>Custo&amp;m Format</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Character Replacement</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QCheckBox" name="spaceCheck" >
         <property name="toolTip" >
          <string>If checked, convert spaces to underscores.</string>
         </property>
         <property name="text" >
          <string>&amp;Replace Spaces with Underscores</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="asciiCheck" >
         <property name="toolTip" >
          <string>If checked, replace characters that are unavailable in the 7-bit ASCII code.</string>
         </property>
         <property name="text" >
          <string>Restrict to &amp;ASCII</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignoreTheCheck" >
         <property name="toolTip" >
          <string>If checked, postfix artists' names starting with 'The' with ', The'.</string>
         </property>
         <property name="text" >
          <string>I&amp;gnore 'The' in Artist Names</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="vfatCheck" >
         <property name="toolTip" >
          <string>If checked, replace characters that are incompatible with MS-DOS/VFAT file systems.</string>
         </property>
         <property name="text" >
          <string>VFAT Safe &amp;Names</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="textLabel1" >
           <property name="text" >
            <string>Replace</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="regexpEdit" >
           <property name="toolTip" >
            <string>Regular expression</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel2" >
           <property name="text" >
            <string>with</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="replaceEdit" >
           <property name="toolTip" >
            <string>Character string</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>83</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder></zorder>
      <zorder>vfatCheck</zorder>
      <zorder>spaceCheck</zorder>
      <zorder>asciiCheck</zorder>
      <zorder>ignoreTheCheck</zorder>
      <zorder>verticalSpacer</zorder>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="previewBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Destination Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <widget class="QLabel" name="previewText" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>This is what the file names will look like after renaming.</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer name="spacer2_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>347</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="overwriteCheck" >
          <property name="toolTip" >
           <string>If checked, overwrite files of the same name without asking.</string>
          </property>
          <property name="text" >
           <string>Overwrite &amp;Destination</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0" />
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kcombobox.h</include>
  <include location="local" >klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
